Resources/Prototypes/Tiles/floors.yml

```diff
@@ -1887,6 +1905,15 @@
   mobAcceleration: 2
   itemDrop: FloorTileItemAstroIce
 
+- type: tile
+  id: FloorAstroSnow
+  name: tiles-astro-snow
+  parent: FloorSnow
+  baseTurf: Plating
+  isSubfloor: false
+  deconstructTools: [ Prying ]
```

Comment:
Why does astro-snow get to be pried, but astro-grass has to be cut? Shouldn't this require shoveling?

Reply:
I tried shovels and welders, but unfortunately they no worky